codebase/config/sync/group.role.study-contributor.yml

```diff
@@ -16,7 +16,9 @@ permissions:
   - 'access group_node overview'
   - 'update any group_node:study entity'
   - 'update own group_node:study entity'
+  - 'view all group_node:study entity revisions'
```

Comment:
does this give a contributor access to revisions of other studies they should not see?

Reply:
I noted that naming of the permission in the yml file but in the front-end, I have enabled only the following two permissions- 

- Revisions: View full version history
- Revisions: View specific entity revisions

Reply:
I have checked if contributor can access other group studies/revisions and they can't.